scripts/generate-key

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env python
 
+# Run example
+# ./scripts/show-verify-key ed25519 0 xjKlejLcLyTQg7Fxy/XGopUeF3W/l3/cRgpFe+edi0E
```

Comment:
I think this is completely bogus :/

Reply:
```
./scripts/show-verify-key ed25519 0 xjKlejLcLyTQg7Fxy/XGopUeF3W/l3/cRgpFe+edi0E
signing key: ed25519 0 xjKlejLcLyTQg7Fxy/XGopUeF3W/l3/cRgpFe+edi0E 
verify key: 3tDie4gcIVBSNrCsOaUv5bQ7hQFsCyg73/nNsWqszsU
```

As in the information is incorrect or?

Reply:
this is a different script... show-verify-key doesn't exist on this branch afaict

Reply:
Oh! Indeed, this was meant to go in the other script.

Reply:
https://github.com/matrix-org/sydent/pull/111